docs/UserGuide.md

```diff
@@ -102,13 +121,23 @@ ___
 
 _This command adds a new client to your client list._
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS k/NEXTOFKIN d/DESCRIPTION [t/TAG]…`
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS k/NEXT_OF_KIN d/DESCRIPTION [t/TAG]…`
 
-- Name being added must be unique
-- All parameters _except tag_ are compulsory
+Parameters:
+
+| Parameter    | Description                                        | Remarks                               |
+|--------------|----------------------------------------------------|---------------------------------------|
+| NAME         | Name of client that you want to add                | Name must be unique and is compulsory |
+| PHONE_NUMBER | Phone Number of client that you want to add        | Compulsory                            |
+| EMAIL        | Email of client that you want to add               | Compulsory                            |
+| ADDRESS      | Address of client that you want to add             | Compulsory                            |
+| NEXT_OF_KIN  | Next of Kin details of client that you want to add | Compulsory                            |
+| DESCRIPTION  | Description of client that you want to add         | Compulsory                            |
```

Comment:
We can consider making Parameters into a dropdown menu so that users can reference if they want and if not, they are not overwhelmed by a huge table.